lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -215,4 +216,10 @@ defmodule ExUnit.CLIFormatter do
       _ -> 80
     end
   end
+
+  defp print_logs(""), do: nil
+
+  defp print_logs(output) do
+    IO.puts(["The following output was logged:\n" | output])
```

Comment:
Can we indent these so it is easier to differentiate between these logs and console?